tinytga/src/lib.rs

```diff
@@ -289,7 +289,7 @@ impl<'a> Iterator for TgaIterator<'a> {
 
         self.x += 1;
 
-        if self.x >= self.tga.width().into() {
+        if self.x >= self.tga.width() as u32 {
```

Comment:
Please remove the changes to `tinytga` because they aren't related to this PR.

Reply:
In fact the 3 changes to `tinytga` are related to this PR. When the `fixed` crate is used, some of those types can't be inferred anymore. Here is the error you get with this code with the `fixed` crate in use:

```
error[E0283]: type annotations needed for `u32`
   --> tinytga/src/lib.rs:292:19
    |
250 |         let mut pixel_value = {
    |             --------------- consider giving `pixel_value` a type
...
292 |         if self.x >= self.tga.width().into() {
    |                   ^^ cannot infer type for type `u32`
    |
    = note: cannot satisfy `u32: core::cmp::PartialOrd<_>`

error: aborting due to previous error
```

Reply:
OK, interesting, I didn't expect that. Can you change this to:
```suggestion
        if self.x >= u32::from(self.tga.width()) {
```
which is a little safer than a cast if the return type of `tga.with` ever changes.

Reply:
OK.